<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UClassDescriptionDisplay</class>
 <widget class="QWidget" name="UClassDescriptionDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,4,0">
   <item>
    <widget class="QLabel" name="labelClassName">
     <property name="text">
      <string>Class Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelHeader">
     <property name="text">
      <string>Header (brief description)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditHeader"/>
   </item>
   <item>
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Description (detailed description)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditDescription"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
